PhysicsTools/NanoAOD/python/jetsAK4_Puppi_cff.py

```diff
@@ -82,7 +82,6 @@
         nSVs = Var("?hasOverlaps('vertices')?overlaps('vertices').size():0", int, doc="number of secondary vertices in the jet"),
         btagDeepB = Var("?(bDiscriminator('pfDeepCSVJetTags:probb')+bDiscriminator('pfDeepCSVJetTags:probbb'))>=0?bDiscriminator('pfDeepCSVJetTags:probb')+bDiscriminator('pfDeepCSVJetTags:probbb'):-1",float,doc="DeepCSV b+bb tag discriminator",precision=10),
         btagDeepFlavB = Var("bDiscriminator('pfDeepFlavourJetTags:probb')+bDiscriminator('pfDeepFlavourJetTags:probbb')+bDiscriminator('pfDeepFlavourJetTags:problepb')",float,doc="DeepJet b+bb+lepb tag discriminator",precision=10),
-        btagCSVV2 = Var("bDiscriminator('pfCombinedInclusiveSecondaryVertexV2BJetTags')",float,doc=" pfCombinedInclusiveSecondaryVertexV2 b-tag discriminator (aka CSVV2)",precision=10),
```

Comment:
Why in `PhysicsTools/NanoAOD/python/jetsAK4_Puppi_cff.py` you are deleting this while in `PhysicsTools/NanoAOD/python/jetsAK4_CHS_cff.py` you are using the modifier?
I think the era modifier approach should be used, otherwise here you are removing `btagCSVV2` also for Run1, Run2...